src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_NmTokens.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+using System.Xml.XPath;
+
+namespace System.Xml.Tests
+{
+    #nullable enable
+    public class TC_SchemaSet_NmTokens : TC_SchemaSetBase
+    {
+        private void TestSchemaCompile(string fileName, bool negative)
+        {
+            string xsd = Path.Combine(TestData._Root, fileName);
+            XmlSchemaSet s = new XmlSchemaSet();
+            int numevents = 0;
+            s.ValidationEventHandler += (sender, args) => {
+                Assert.False(args.Severity == XmlSeverityType.Warning);
```

Comment:
Use `Assert.Equal` if always expecting a specific value oterwise use `Assert.NotEqual` since the error message will be better if this fails